scripts/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+echo "Restoring cached BigQuery schemas"
+
+SCHEMAS_DIR=/tmp/scio-bigquery-$USER/.bigquery/
+
+mkdir -p $SCHEMAS_DIR
+cp -v scripts/bigquery/* $SCHEMAS_DIR
+
+echo "Adding BigQuery flags"
+echo "-Dbigquery.project=dummy-project" >> .jvmopts
```

Comment:
I like the idea of avoiding setting the sys prop when running sbt. The downside of this is that contributors running this script will have `.jvmopts` staged for commit and it can lead to unwanted changes.

Maybe we need to rethink the default value of `bigquery.project` and have a red warn if this setting is used; or try to make it really lazy as you pointed out.

Reply:
What about this:
We move these examples that require GCP creds to a separate source dir, e.g. `/src/{main,test}/scala-gcp`, and conditionally include them in sbt if `bigquery.project` is set?